### PR TITLE
🔨 Add Header

### LINKED ISSUE #1: Add 'header' component

### PR DESCRIPTION
**Pull request by JuKahl**
- 💄 Add global colors & fonts
- 🍱  Add icons
- 🔨  Add profile picture
- 🔨  Add sub-navigation
- 🔨  Add navigation/back button
- 💄 Add styling
- 🔨  Add header
- ⚰️  Removed unnecessary code
- ♻️  Refactor icon link

[Storybook deployment](https://king-of-the-header-8i6ypa022hs.herokuapp.com/storybook/index.html?path=/story/common-header--primary)

### CODE REVIEW

File: components/header/Header.module.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  grid-template-areas: "navButton label profile";
+  background-color: var(--primary-color);
+}
+
+.menuBtn {
+  grid-area: navButton;
+  border-color: transparent;
+  background-color: transparent;
+  height: 2rem;
+  width: 2rem;
```
Comment (mewiemers) [inline components/header/Header.module.css:13]: maybe use padding to set the size of `menuBtn`

**Review by DNaunin — APPROVED** [review]
LGTM! 🚀

File: components/header/HeaderSubNav.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  subNavOne: string;
+  subNavTwo: string;
+};
+
+export default function SubNav({ subNavOne, subNavTwo }: Navigation) {
+  return (
+    <section className={styles.subNav}>
+      <button
+        className={styles.navOne}
+        onClick={() => console.log("left button: clicked")}
```
Comment (mewiemers) [inline components/header/HeaderSubNav.tsx:14]: do you need this `console.log` anymore?

**Review by mewiemers — COMMENTED** [review]
GJ!😎⚽

File: components/header/HeaderSubNav.tsx (hunk 2 above)
Comment (JuKahl) [reply components/header/HeaderSubNav.tsx:14]: I have left it as a placeholder for the later linkings.

**Review by RonjaMeinecke — APPROVED** [review]
YAY 🚀💥

**Pull request merged**